code/modules/clothing/outfits/ert/inteq_ert.dm

```diff
@@ -0,0 +1,46 @@
+/datum/outfit/centcom/ert/inteq
+	name = "ERT - Inteq Rifleman"
+
+	mask = /obj/item/clothing/mask/gas/sechailer/inteq
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses/inteq
+	uniform = /obj/item/clothing/under/syndicate/inteq
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/inteq
+	suit_store = /obj/item/gun/ballistic/automatic/assault/ak47/inteq
```

Comment:
the ak is pretty powerful. It'd be nice to have a semi auto rifle of some flavor that isn't a DMR.